### PR TITLE
Change the way we specify ip addresses in ansible

### PR DESCRIPTION
**Pull request by spuiuk**
This makes it easier to reuse the ip address list for other purposes.

Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>

### CODE REVIEW

**Comment by spuiuk** [comment]
Sorry, I thought I had already pushed this through. But looks like I didn't create a PR.

**Comment by spuiuk** [comment]
Third part of #25

File: vagrant/ansible/cluster-vars.yml [hunk 1]
```diff
@@ -45,9 +45,15 @@ gluster_cluster_transport: 'tcp'
...
-gluster_features_ctdb_publicaddr: '192.168.123.10/24 eth2,192.168.123.11/24 eth2'
-gluster_features_ctdb_volume: ctdb
+ctdb_network_private_interfaces:
+  - "192.168.122.100"
+  - "192.168.122.101"
+ctdb_network_public_interfaces:
+  - "192.168.123.10"
+  - "192.168.123.11"
+ctdb_network_public_interface_subnet_mask: "24"
+ctdb_network_public_interface_name: "eth2"
```
Comment (obnoxxx) [inline vagrant/ansible/cluster-vars.yml:55]: Just note, that in theory we might have multiple different public addresses with different net masks and different interface names... But of course we have control over it here...

**Comment by spuiuk** [comment]
The variable list which was replaced was only used by the gluster.features script. Hence I saw it fit to generate it on the fly only for that particular role. Easier to use a list of ip addresses to generate the gluster_features_ctdb_public address than the other way around.
The list of ip addresses is also consumed by the client.prep and client.setup roles which are being considered.

We could consider using a dictionary to list the ipaddress/ethernet interface name combination to describe the combination required for the public interfaces. But do we want to make it more complicated than it needs to be at this stage?

**Pull request merged**